website/content/docs/examples/lens-app-iq-in-kind.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Lens AppIQ in Kind"
+draft: false
+---
+
+This blueprint bootstraps a kind cluster and installs Lens AppIQ.
+
+### Bootsrap a Kind cluster with Lens AppIQ
+
+#### Set ENV variables
+
+Set admin user's email and password.
+```shell
+export LAIQ_USER=<Admin User Email>
+export LAIQ_PASSWORD=<Admin user password>
+```
+
+#### Apply the blueprint
+
+Our [example blueprint](https://raw.githubusercontent.com/Mirantis/boundless/main/blueprints/lensappiq/lensappiq-kind-blueprint.yaml) for Lens AppIQ will bootstrap a `kind` cluster, install `Boundless Operator`, install addons from the blueprint. Once you've downloaded the blueprint, apply it with `bctl`:
```

Comment:
We should add similar link to the blueprint of lensappiq-k0s-lima-blueprint.yaml and lensappiq-k0s-blueprint.yaml, I left the comment in respective documents